app/controllers/application_controller.rb

```diff
@@ -1,3 +1,5 @@
 class ApplicationController < ActionController::Base
   include Pagy::Backend
+
+  skip_before_action :verify_authenticity_token
```

Comment:
After I added the action buttons in edit page, when I wanted to edit the page I got this error: Can't verify CSRF token authenticity. As i understood is because the end of submit buton does a POST method and throw the action buttons I use POST and DELETE methods. This was the way I foud to fix the issue.
https://plata.news/blog/cant-verify-csrf-token-authenticity/

Reply:
It is weird, as we had CRUD functionalities in the UI before, and this wasn't needed 🤔 
I suppose that after you remove the 3 custom endpoints, and use the #update, we won't need this

Reply:
If I get rid of the buttons where I specify the method (PUT, DELETE) it works without skiping the authenticity token.

Reply:
Fixed in my commit